### ISSUE TITLE
aws_ecs_task_definition not passing initProcessEnabled 

### ISSUE THREAD

**Issue by ojundt** [terraform-providers/terraform-provider-aws#2174]
Hi,

There is a new linux parameter `initProcessEnabled` for ECS task container definitions: 
https://aws.amazon.com/about-aws/whats-new/2017/11/amazon-ecs-adds-support-for-docker-device-and-init-flags-in-container-task-definitions/
http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_LinuxParameters.html

I can set the parameter in the JSON config using the AWS web interface but applying it through terraform does not work. The setting is being ignored without a warning or error message.

### Terraform Version
Terraform v0.10.8
AWS Provider version 1.2.0

### Affected Resource(s)
`aws_ecs_task_definition`

### Terraform Configuration Files
```hcl
resource "aws_ecs_task_definition" "foobar" {
  family = "foobar"
  container_definitions = "${file("container_definitions.json")}"
}
```

`container_definitions.json`:
```json
[{
  "name": "foobar",
  "image": "${image}",
  "memory": 192,
  "portMappings": [{
    "containerPort": 3000
  }],
  "linuxParameters": {
    "initProcessEnabled": true
  }
}]

```

### Expected Behavior
`initProcessEnabled` should be `true`

### Actual Behavior
`initProcessEnabled` is `null` and apply goes through without a warning or error message.

### Steps to Reproduce
Apply a task with above mentioned container definition.

### Important Factoids
I tried other `linuxParameters` like `capabilities` as mentioned in #1801 and they work. Only `initProcessEnabled` seems to be ignored.

**Comment by ojundt** [comment]
Just checked the recent commits and a bump of the AWS sdk version fixes it (b416e9467ca882b5d1637c7cbef4fae426b92e3c)

**Comment by avivnoy** [comment]
Thaks for the update!
Is it solve your zombie issues?
